app/src/backend/UserService.ts

```diff
@@ -6,6 +6,7 @@ import type { City } from "@/models/City";
 import type { Inventory } from "@/models/Inventory";
 import type { User } from "@/models/User";
 import type { Includeable } from "sequelize";
+import { InventoryValue } from "@/models/InventoryValue";
```

Comment:
You can do `import type` here as well, so the class isn't added to the final JS output since it's only used during type checking.

Reply:
route no longer exists deletion